_data/link.yml

```diff
@@ -42,9 +42,8 @@ Bin:
   link: bin.html
 BinParams:
   link: "bin.html#bin-parameters"
-FilterOperand:
-  name: Filter
-  link: "transform.html#filter"
+"LogicalOperand<Predicate>":
+  link: "transform.html#predicate"
```

Comment:
do you want to give this a better name?

Reply:
This is good since there is LogicalOperand<String> too


So having generic should allow us to write about it only once

Reply:
I see. Go ahead and merge then.